tools/cfp-vote-counter/main.py

```diff
@@ -42,31 +43,32 @@ def __init__(self, event: str, env: str):
         self.voting_end = Decimal(VOTING_PERIOD[event][1].timestamp() * 1000)
 
 
+def fetch_votes(conf: Config) -> DynamoResponse:
+    dynamo = boto3.resource("dynamodb")
```

Comment:
region_name指定したいという意図なので、（定数上にもっていった方が良いとかの）細かい実装はお任せします 🙇 

```suggestion
    dynamo = boto3.resource("dynamodb" ,region_name='ap-northeast-1')
```

Reply:
aws sso loginするとdefault regionが指定されるから良いかなと思っていたのですが、複数region扱っていてdefault regionが別のところを向いている場合に困ってしまう、と理解しました。

Boto3Configクラスを追加し、引数から指定できるようにしました。